integration-cli/docker_cli_save_load_test.go

```diff
@@ -167,6 +167,20 @@ func (s *DockerSuite) TestSaveAndLoadRepoFlags(c *check.C) {
 	c.Assert(before, checker.Equals, after, check.Commentf("inspect is not the same after a save / load"))
 }
 
+func (s *DockerSuite) TestSaveWithNoExistImage(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	name := "test-save-image"
+	dockerCmd(c, "run", "--name", name, "busybox", "true")
```

Comment:
Oh, now I see what you meant; but is it necessary? All we need is an image name that we know doesn't exist, so creating it first, then removing it shouldn't be needed?